docs/03 - Implementing the Anchor Server/Communication/Platform API.yml

```diff
@@ -85,14 +84,9 @@ paths:
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  records:
-                    type: array
-                    items:
-                      $ref: './schemas.yml#/components/schemas/Transaction'
-                  cursor:
-                    type: string
+                oneOf:
+                  - $ref: './schemas.yml#/components/schemas/AllTransactionsSEP24'
```

Comment:
How do we return the list of transactions of all seps?

Reply:
It's the same endpoint. I just wanted to show that get transaction can return either SEP-31 or SEP-24